src/utils/getBoundsFromGeoServer.js

```diff
@@ -0,0 +1,32 @@
+import X2JS from 'x2js';
+
+export const getBoundsFromWmsLayer = (wmsSlug, wmsUrl, value, valueChanged) => {
+    // Use /proxy/ to avoid the request is blocked by CORS policy
+    // Note: in production, it will probably be working without the prefix /proxy/
+    const proxyUrl = `/proxy/${wmsUrl}`;
+
+    fetch(`${proxyUrl}/?request=getCapabilities`)
+        .then(response => {
+            if (!response.ok) throw new Error('Oops! Cannot get data from the GeoServer');
```

Comment:
This error should be handled somehow (e.g. show an error next to the button).

Reply:
I have a catch at line 30 that give user an alert message.

Reply:
Ah right, I thought it said console.error instead of alert. But alert() looks very ugly and the error message is likely to be hard to understand for a user, I think something like a short message "failed to get extent from geoserver" in red next to the button would be better.

Reply:
Nice! I removed the alert() and added an error message next to the button :)